src/SFA.DAS.Roatp.Functions/ApplicationExtract.cs

```diff
@@ -46,12 +55,122 @@ public ApplicationExtract(ILogger<ApplicationExtract> log, ApplyDataContext appl
             }
         }
 
+        private static List<OrganisationPersonnel> ExtractOrganisationPersonnel(List<SubmittedApplicationAnswer> answers, Guid organisationId, string questionId, PersonnelType personnelType)
+        {
+            var submittedAnswersOrganisationPersonnel = answers.Where(answer => answer.QuestionId == questionId).GroupBy(a => a.RowNumber).ToList();
+            var organisationPersonnel = new List<OrganisationPersonnel>();
+
+            if (personnelType == PersonnelType.PersonInControl && questionId == QuestionIdSoleTrade)
+            {
+                var submittedAnswersOrganisationNameSoleTrade = answers.Where(answer => answer.QuestionId == QuestionIdOrganisationNameSoleTrade);
+                if (submittedAnswersOrganisationPersonnel.Count > 0)
+                {
+                    foreach (var person in submittedAnswersOrganisationPersonnel)
+                    {
+                        var orgPersonnel = new OrganisationPersonnel
+                        {
+                            OrganisationId = organisationId,
+                            PersonnelType = (int)personnelType,
+                            Name = submittedAnswersOrganisationNameSoleTrade.FirstOrDefault().Answer
+                        };
+                        foreach (SubmittedApplicationAnswer record in person)
+                        {
+                            var dobArray = record.Answer.Split(",");
+                            orgPersonnel.DateOfBirthMonth = byte.Parse(dobArray[0]);
+                            orgPersonnel.DateOfBirthYear = int.Parse(dobArray[1]);
+                        }
+                        organisationPersonnel.Add(orgPersonnel);
+                    }
+                }
+                else
+                {
+                    var orgPersonnel = new OrganisationPersonnel
+                    {
+                        OrganisationId = organisationId,
+                        PersonnelType = (int)personnelType,
+                        Name = submittedAnswersOrganisationNameSoleTrade.FirstOrDefault().Answer
+                    };
+                    organisationPersonnel.Add(orgPersonnel);
+                }
+            }
+            else
+            {
+                foreach (var person in submittedAnswersOrganisationPersonnel)
+                {
+                    var orgPersonnel = new OrganisationPersonnel
+                    {
+                        OrganisationId = organisationId,
+                        PersonnelType = (int)personnelType,
+                    };
+                    foreach (var record in person)
+                    {
+                        switch (record.ColumnHeading)
+                        {
+                            case "Name":
+                                orgPersonnel.Name = record.Answer;
+                                break;
+                            case "Date of birth":
```

Comment:
Have you smoke tested this? |I thought the data was of the form "Month Year", and not sure about the conversion working.  If you've smoke tested and all ok, just resolve this conversation